demo-notebooks/guided-demos/preview_nbs/0_basic_ray.ipynb

```diff
@@ -62,6 +62,7 @@
     "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\" unless you specify the local queue manually below\n",
     "cluster = Cluster(ClusterConfiguration(\n",
     "    name='raytest',\n",
+    "    head_gpus=1, # For GPU enabled workloads set the head_gpus and num_gpus\n",
```

Comment:
```suggestion
    "    head_gpus=0, # For GPU enabled workloads set the head_gpus and num_gpus\n",
```

Reply:
Added head_gpus=0 by default as we have num_gpus=0 here as well